cfy_manager/components/haveged/haveged.py

```diff
@@ -38,4 +37,10 @@ def start(self):
 
 def using_systemd_haveged():
     # On more complete installs of RHEL/Centos, haveged may already be running
-    return service.SystemD().is_installed('haveged')
+    if service.SystemD().is_installed(HAVEGED):
+        logger.notice('Using system haveged')
+        return True
+    if service.SystemD().is_active(HAVEGED):
+        logger.notice('System haveged active but disabled. Stopping it... ')
+        service.SystemD().stop(HAVEGED)
```

Comment:
Doesn't this fail in containers without cap sysadmin?

Reply:
From out of band discussion, this is probably not a warranted concern. If it is, we'll put out the fire when it erupts.